File: benchmarks/vm/02_loop_sum.rb
t0 = Process.clock_gettime(Process::CLOCK_MONOTONIC)
total = 0
1_000_000.times { |i| total += i }
puts "sum = #{total}"
puts "BENCH_RESULT: #{((Process.clock_gettime(Process::CLOCK_MONOTONIC) - t0) * 1000).to_i} ms"

File: benchmarks/vm/02_loop_sum.lua
local t0 = os.clock()
local total = 0
for i = 0, 999999 do
    total = total + i
end
print("sum = " .. tostring(total))
print("BENCH_RESULT: " .. tostring(math.floor((os.clock() - t0) * 1000)) .. " ms")
